### ISSUE TITLE
Video recording crashing

### ISSUE THREAD

**Issue by ayr-ton** [zhuowei/Xenologer#4]
When said to record a video in Asus TF101 the camera app crash.

**Comment by ChristianFJung** [comment]
Yes it does. I think maybe we should just try it differently like this.

Ok glass take a video

Glass opens up the camera app and starts recording.

Would that be easier @zhuowei
